mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AbstractAttachmentMapperTest.java

```diff
@@ -60,46 +65,113 @@ public void tearDown() throws MailboxException {
     }
 
     @Test
-    public void addShouldWork() throws IOException {
+    public void putShouldWork() throws IOException {
         // Given
-        String blobId = "blobId";
-        String expected = "content";
+        InMemoryAttachmentId blobId = InMemoryAttachmentId.of(1);
+        String expectedContent = "content";
+        Content content = new ByteContent(expectedContent.getBytes(Charsets.UTF_8));
```

Comment:
1. Section is a test fixture, so streamable content does not really matter much here I suppose.
2. Anyway, the "Content" interface used in Attachment is indeed not very appropriate.
Unfortunately found no implementation that can manage binary attachment.
We may need to add an abstraction to type binary content of attached files (and replace current Content) if none match in existing james codebase.
Antoine originally used InputStream which was assumed too "primitive", but we may reconsider. Suggestions, anyone ?